services/movie.py

```diff
@@ -0,0 +1,31 @@
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> list[Movie]:
+    movies = Movie.objects.all()
+
+    if genres_ids:
+        movies = movies.filter(genres__id__in=genres_ids).distinct()
+
+    if actors_ids:
+        movies = movies.filter(actors__id__in=actors_ids).distinct()

```

Comment:
The current filtering logic uses an 'AND' operation when filtering by genres and actors. If the intention is to match either condition, consider using `Q` objects for an 'OR' operation to allow more flexible querying.